pkg/util/kube/pvc_pv.go

```diff
@@ -69,7 +70,7 @@ func DeletePVAndPVCIfAny(ctx context.Context, client corev1client.CoreV1Interfac
 		}
 	}
 
-	if err := client.PersistentVolumeClaims(pvcNamespace).Delete(ctx, pvcName, metav1.DeleteOptions{}); err != nil {
+	if err := EnsureDeletePVC(ctx, client, pvcName, pvcNamespace, ensureTimeout); err != nil {
```

Comment:
If the ensureTimeout is set to 0, what is the behavior of `wait.PollUntilContextTimeout`?
Does the function fail immediately, or return no matter what happens?

Reply:
It could abort the wait immediately, but it will return a timeout error which is not expected.
I have modified to explicitly check the ensureTimeout==0 case.